README.md

```diff
@@ -13,6 +13,11 @@ See the [Documentation](https://brain-score_language.readthedocs.io) for more de
 Brain-Score is made by and for the community. To contribute,
 please [send in a pull request](https://github.com/brain-score/language/pulls).
 
+## Alpha status
+This repository is under active development, and should be considered to be in an alpha status.
```

Comment:
alpha or pre-alpha?

Reply:
I thought alpha was accurate; I think we have something usable already